lib/src/models/data.ts

```diff
@@ -38,6 +38,7 @@ export interface DataLayer<T> {
     setOIDCProviderMetaData(oidcProviderMetaData: OIDCProviderMetaData): Promise<void>;
     setConfigData(config: AuthClientConfig<T>): Promise<void>;
     setTemporaryData(data: TemporaryData): Promise<void>;
+    getPKCECode(sessionId: string): Promise<PKCECode>;
```

Comment:
Why do we need this?

Reply:
A part came from an old PR. I think its better to close this and send a fresh PR